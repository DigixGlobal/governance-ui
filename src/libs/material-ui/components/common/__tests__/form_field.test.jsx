import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import FormField from '../form_field';

const formData = {
  TestField: 'test value',
};

describe('<FormField />', () => {
  test('renders correctly', () => {
    const component = shallow(
      <FormField
        formChange={jest.fn}
        formData={formData}
        name="TestField"
      />,
    );

    expect(toJson(component)).toMatchSnapshot();
  });
});
