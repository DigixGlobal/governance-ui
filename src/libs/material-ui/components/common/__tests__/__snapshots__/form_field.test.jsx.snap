// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FormField /> renders correctly 1`] = `
<FormField>
  <Input
    name="TestField"
    onChange={[Function]}
    type="text"
    value="test value"
  />
</FormField>
`;
